server/sql.go

```diff
@@ -138,6 +140,49 @@ func (s *Server) registerSQLRoutes(g *echo.Group) {
 			return json.Marshal(rowSet)
 		}()
 
+		{
+			errMessage := ""
+			if err != nil {
+				errMessage = err.Error()
+			}
+
+			bytes, err := json.Marshal(api.ActivitySQLQueryHistoryPayload{
+				Statement:    exec.Statement,
+				Duration:     time.Now().UnixMilli() - startTms,
+				InstanceID:   exec.InstanceID,
```

Comment:
We already saved instanceID in the container_id. I would suggest to add instanceName. Otherwise, we will run into N+1 issue when listing the query history (first fetching the instance Id list, and then fetching N instance name for each of the instance id)